src/app/core/navbar/navbar.component.html

```diff
@@ -167,10 +167,10 @@ <h5 class="modal-title" id="changelog-label">Changelog</h5>
           <small
             >Note: Whenever we release an update to the 'signing' page, we may,
             at our discretion, allow you to send <em>another</em> message to
-            each of your friends' virtrolios! This ensures that you will be able
-            to enjoy any new features that we develop. Keep an eye on this
-            changelog as we will mention here if we ever do give you another
-            message opportunity.</small
+            each of your friends' virtrolios as if it were a new year! This
+            ensures that you will be able to enjoy any new features that we
+            develop. Keep an eye out for versions with a 💌 emoji next to them
+            to see if you're allowed to spread some more Virtrolio joy!</small
```

Comment:
I like the wording before but we also do need to be extra clear to make sure people don't miss it
```suggestion
            to see if you're allowed to send an extra message!</small
```